SpatialGDK/Source/SpatialGDK/Private/SpatialGDKSettings.cpp

```diff
@@ -132,6 +132,15 @@ void USpatialGDKSettings::PostInitProperties()
 	}
 	UE_LOG(LogSpatialGDKSettings, Log, TEXT("Result types are %s."), bEnableResultTypes ? TEXT("enabled") : TEXT("disabled"));
 
+	UE_LOG(LogSpatialGDKSettings, Log, TEXT("Handover is %s."), bEnableHandover ? TEXT("enabled") : TEXT("disabled"));
```

Comment:
For my knowledge:
We construct many FStrings here that are the same value. Do we rely on compiler optimizing this?

Reply:
No, compiler won't (to the best of my knowledge), optimise this away *when logs are enabled*. Logs will be disabled for any normal test/shipping build, and in that instance they won't be constructed, so it wouldn't have an impact on the end product.